bcda/auth/backend.go

```diff
@@ -135,11 +138,24 @@ func (backend *JWTAuthenticationBackend) RevokeToken(tokenString string) error {
 	db := database.GetDbConnection()
 	defer db.Close()
 
-	err := token.Insert(db)
+	var err error
+
+	if token.Exists() {
+		err = token.Update(db)
+	} else {
+
+		const sqlstr = `INSERT INTO public.tokens (` +
+			`uuid, user_id, value, active` +
+			`) VALUES (` +
+			`$1, $2, $3, $4` +
+			`)`
+
+		_, err = db.Exec(sqlstr, token.UUID, token.UserID, token.Value, false)
```

Comment:
Why not use `token.Insert(db)` here?

Reply:
I tried using that but got the "No rows in result set" error. I don't think Postgres returns anything on insert without `RETURNING...` in the statement, which isn't in the one xo generated.